stratum/hal/lib/tdi/es2k/es2k_node.cc

```diff
@@ -28,24 +28,23 @@ namespace tdi {
 
 // Initialize base class TdiNode members as well
 Es2kNode::Es2kNode(TdiTableManager* tdi_table_manager,
-                 TdiActionProfileManager* tdi_action_profile_manager,
-                 TdiPacketioManager* tdi_packetio_manager,
-                 TdiPreManager* tdi_pre_manager,
-                 TdiCounterManager* tdi_counter_manager,
-                 TdiSdeInterface* tdi_sde_interface, int device_id,
-                 bool initialized, uint64 node_id,
-                 TdiLutManager* tdi_lut_manager)
+                   TdiLutManager* tdi_lut_manager,
+                   TdiActionProfileManager* tdi_action_profile_manager,
+                   TdiPacketioManager* tdi_packetio_manager,
+                   TdiPreManager* tdi_pre_manager,
+                   TdiCounterManager* tdi_counter_manager,
+                   TdiSdeInterface* tdi_sde_interface, int device_id,
+                   bool initialized, uint64 node_id)
     : TdiNode(tdi_table_manager, tdi_action_profile_manager,
               tdi_packetio_manager, tdi_pre_manager, tdi_counter_manager,
-             tdi_sde_interface, device_id, initialized, node_id),
+              tdi_sde_interface, device_id, initialized, node_id),
       pipeline_initialized_(false),
       initialized_(initialized),
       tdi_config_(),
       tdi_sde_interface_(ABSL_DIE_IF_NULL(tdi_sde_interface)),
       tdi_table_manager_(ABSL_DIE_IF_NULL(tdi_table_manager)),
-      tdi_lut_manager_(tdi_lut_manager),
-      tdi_action_profile_manager_(
-          ABSL_DIE_IF_NULL(tdi_action_profile_manager)),
+      tdi_lut_manager_(ABSL_DIE_IF_NULL(tdi_lut_manager)),
+      tdi_action_profile_manager_(ABSL_DIE_IF_NULL(tdi_action_profile_manager)),
```

Comment:
alignment?

Reply:
Alignment should be correct. I ran clang-format on the file.